joplin/templates/wagtailadmin/skeleton.html

```diff
@@ -0,0 +1,78 @@
+{# NOTE: This template was added("copy/pasted") from the wagtail library #}
```

Comment:
This is a helpful note. 

But also;
Is the only change here adding the javascript? 

You could add that using the webpack setup we have. 

Or also, it would be pretty simple to use their hook:
https://docs.wagtail.io/en/v2.8/reference/hooks.html#insert-editor-js

That way we future-proof future updates more by not having to also check all the templates. But if this is temporary it may not matter too much.

Reply:
We've also added our custom templates in here as well as having access to the header for adding style / other dependancies. Also, there is some fun stuff that I may want to look at, like, this where they language tag is in place to be applied, which we haven't done. 
-
But, yes, the skeleton is the primary template, so this one specifically should be kept an eye on when there are any updates. 
-
So, i added some more notes in the actual file.